CHANGES.rst

```diff
@@ -5,7 +5,11 @@ CHANGES
 4.0.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Drop support for Python 3.4.
+
+- Add support for Python 3.7 and 3.8.
+
+- Update PyPy version.
```

Comment:
Some people are insisting that dropping supported Python versions mean we bump the minor version rather than the patchlevel, so this should be `4.1.0 (unreleased)`, with a corresponding version update in `setup.py`.

Reply:
Thanks for the review. I bumped the version accordingly.